docs/subnets/reference-cli-commands.md

```diff
@@ -236,6 +242,43 @@ avalanche subnet import [subnetPath] [flags]
 
 <!-- markdownlint-enable MD013 -->
 
+#### Import from a public network 
+
+The subnet import public command will import a subnet configuration from a running network.
+
+The genesis file should be available from the disk for this to work. 
+By default, an imported subnet will not overwrite an existing subnet with the same name. 
```

Comment:
```suggestion
By default, an imported Subnet will not overwrite an existing Subnet with the same name. 
```
All words Subnet should have first letter upper-cased, except in the context of a command, path or logs. See https://github.com/ava-labs/avalanche-docs/blob/master/style-guide.md#style-guide